server/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist/static/js/bundle.js'));
+
+const PORT = 3101;
+
+app.get('/tiersIncluded', (req, res) => {
+  const dataString = { bundleId: 1 };
+  return db.tiersRequest(dataString)
+  .then(tiersData => {
+    if (tiersData) {
+      return tiersData;
+    } else {
+      res.status(404).send('no such bundleId');
+    }
```

Comment:
I am uncertain why, in your routes, you are returning your data when you are firing a `.then` block right after this sending you data to the endpoint and handling an error here when you have a `.catch`.

Reply:
I see. This is where, should a error be returned from your db, it would be handled.

Reply:
Again, I am uncertain but I think this would not catch your error. Did you check it by making a query to your db that returns an error?